fiona/collection.py

```diff
@@ -87,6 +89,8 @@ def __init__(self, path, mode='r', driver=None, schema=None, crs=None,
         self._crs_wkt = None
         self.env = None
         self.enabled_drivers = enabled_drivers
+        self.ignore_fields = ignore_fields
+        self.ignore_geometry = bool(ignore_geometry)
```

Comment:
Great. Given a collection it will sometimes be good to know if fields have been ignored.